qlib/workflow/expm.py

```diff
@@ -322,18 +333,17 @@ def client(self):
             self._client = mlflow.tracking.MlflowClient(tracking_uri=self.uri)
         return self._client
```

Comment:
I think you might need to sync the returned self._client to the updated default_uri. 
Or you have set the uri in every function in access the client.

Reply:
Fixed Now.